### DISCUSSION TITLE
How I can Delete git:(master) from oh-my-zsh?

### DISCUSSION THREAD

**Discussion by zohoeth** [ohmyzsh/ohmyzsh#10103] [Help] [answered]
Hi 👋 Have some question about start Default theme!

I can't remove this git:(master) on beginning - and I don't use git.

<img width="842" alt="Снимок экрана 2021-08-14 в 11 03 39" src="https://user-images.githubusercontent.com/41522942/129439511-c255af97-e62b-498a-bd6a-5e3f0e2c2781.png">

How I can delete this ?

**Comment by rwmitchell** [comment, marked as answer]
rm -rf .git

**Reply by zohoeth** [reply]
cool 😎 thanks 🙏

**Reply by zohoeth** [reply]
and have some one question
How I can make a name of window look like my work directory for example "~/Desktop"

**Reply by rwmitchell** [reply]
This will depend on the terminal program you are using.
According to: 
```https://en.wikipedia.org/wiki/ANSI_escape_code```

for xterm, you would:
```Xterm allows the window title to be set by ESC ]0;this is the window title BEL.```

which can be more easily written as:
```printf "\e]0;$PWD\a"```
